src/derivation.ts

```diff
@@ -221,6 +221,9 @@ const modes = Object.freeze({
   filecoin: {
     overridesDerivation: "44'/461'/0'/0/<account>",
   },
+  hederaBip44: {
+    overridesDerivation: "44/3030/0/0/<account>",
```

Comment:
44'/3030' ?

Reply:
I'm not sure why it's the case, but `44'/3030'/0/0/<account>` doesn't seem to work or when trying to add an account, will not return the associated list of accounts. `44/3030'/0/0/<account>` _does_ work, but I'm not sure how much of an improvement it would be over what's currently there.